prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -3,14 +3,14 @@ groups:
   rules:
   - alert: CinderBackendShardLowOvercommitWarning
     expr: >
-      sum(cinder_max_oversubscription_ratio - cinder_overcommit_ratio) by (backend, shard) < .5
+      sum(cinder_free_until_overcommit_gib) by (region, shard) / 1024 + sum(cinder_free_until_overcommit_gib / cinder_total_capacity_gib * 100 < 10) by (region, shard) * 0
```

Comment:
I think this needs a `backend` in the `by (region, shard)`, too.

Reply:
ok updated